Source/Core/Core/IOS/ES/Formats.h

```diff
@@ -68,6 +68,7 @@ struct TMDHeader
   u8 tmd_version;
   u8 ca_crl_version;
   u8 signer_crl_version;
+  u8 is_vwii;
```

Comment:
This used to be a padding byte, right? Might be worth adding a comment to explain the value is only meaningful for vWii TMDs.

(also, it'd be nice to add an offset assertion for `ios_id` just to make sure this doesn't shift stuff down)

Reply:
Yeah, used to be an always 0 padding byte. Nintendo set this byte to 1 for vWii TMDs.
Will add an assert + comment.